enumeratum-core/src/test/scala-2/enumeratum/EnumSpecCompat.scala

```diff
@@ -0,0 +1,14 @@
+package enumeratum
+
+private[enumeratum] trait EnumSpecCompat { _: EnumSpec =>
+  describe("Scala2 in") {
+    it(
```

Comment:
Just want to double check that this is indeed no longer failing in Scala 3; kinda odd.

Reply:
https://github.com/lloydmeta/enumeratum/pull/349/files#diff-389a40c956380c6b16f190d2f6751274a77c695caf7aff37969aae756679cd97R3